src/home/Home.jsx

```diff
@@ -5,19 +5,19 @@ export default function Home() {
 
     return (
         <React.Fragment>
-        <section id="home">
-        <div className="header-text">
-        <h1>Welcome to<br/> Bridge-in-Tech</h1>
-        <p>Bridge-In-Tech is an application inspired by the existing
-        AnitaB.org Mentorship System. It encourages organizations to collaborate
-        with the mentors and mentees on mentoring programs. Through Bridge-In-Tech,
-        an organization can offer a mentorship program to a mentor and a mentee that is customised
-        to meet the needed skills set within its organisation while providing a safety and supportive
-        environment for these mentor/mentee to work in.
-        </p>
-        </div>
-        </section>
-        <GetInTouch />
+            <section id="home">
+            <div className="header-text">
+            <h1>Welcome to<br/> Bridge-in-Tech</h1>
+            <p>Bridge-In-Tech is an application inspired by the existing
+            AnitaB.org Mentorship System. It encourages organizations to collaborate
+            with the mentors and mentees on mentoring programs. Through Bridge-In-Tech,
+            an organization can offer a mentorship program to a mentor and a mentee that is customised
+            to meet the needed skills set within its organisation while providing a safety and supportive
+            environment for these mentor/mentee to work in.
+            </p>
+            </div>
+            </section>
+            <GetInTouch />
```

Comment:
```suggestion
<section id="home">
    <div className="header-text">
        <h1>Welcome to<br /> Bridge-in-Tech</h1>
        <p>Bridge-In-Tech is an application inspired by the existing
            AnitaB.org Mentorship System. It encourages organizations to collaborate
            with the mentors and mentees on mentoring programs. Through Bridge-In-Tech,
            an organization can offer a mentorship program to a mentor and a mentee that is customised
            to meet the needed skills set within its organisation while providing a safety and supportive
            environment for these mentor/mentee to work in.
        </p>
    </div>
</section>
```

This is what @Rahulm2310 wants to say @shades-7

Reply:
yup! Thanks @codesankalp

Reply:
hey @mtreacy002  can you review my PR?

Reply:
will look into it 😉